tests/utils.go

```diff
@@ -193,19 +244,43 @@ func createThinStorageClass() {
 }
 
 // VerifyLVMVolume verify the properties of a lvm-volume
-func VerifyLVMVolume() {
+// expected_vg is supposed to be passed only when vgpatten was used for scheduling.
+// If its volgroup in sc then we can just match volgroup with lvmvolume's vg field.
+func VerifyLVMVolume(no_provision bool, expected_vg string) {
```

Comment:
```suggestion
func VerifyLVMVolume(expect_provisioned bool, expected_vg string) {
```

Accordingly, following checks would need to be 
```
if !expect_provisioned {
} else {
}
```

Reply:
Done